feat: text-utilities

harshitadaswani opened the pull request:
added small text, grey text , center text

harshitadaswani committed:
feat: text-utilities

added small text, grey text , center text

JantuDeb reviewed (approved):
Looks good to me.

ajit-sharmaGH reviewed (approved):
Everything Looks cool 😎

harshitadaswani committed:
feat: added code snippets

- fixed responsiveness of main content
- added utility classes for center and grey text
- refactored css of example box div
